cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -504,12 +503,10 @@ private void updateExplore(DatasetId datasetInstance, DatasetInstanceConfigurati
     try {
       exploreFacade.updateExploreDataset(datasetInstance.toId(), oldSpec, newSpec);
     } catch (Exception e) {
-      String msg = String.format("Cannot enable exploration of dataset instance %s of type %s: %s",
-                                 datasetInstance, creationProperties.getProperties(), e.getMessage());
-      LOG.error(msg, e);
+      LOG.error("Cannot update explore for dataset instance {} with old properties {} and new properties {}: {}",
```

Comment:
ditto. I can remove all these. Just that my intention was to fix the message, not the original behavior.